flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/sink/StoreSink.java

```diff
@@ -123,22 +191,20 @@ public StoreSink(
                     });
         }
 
-        return new StoreGlobalCommitter<>(
-                commit, fileStore.newExpire(), fileCommitSerializer(), lock, overwritePartition);
+        return new StoreGlobalCommitter(commit, fileStore.newExpire(), lock, overwritePartition);
     }
 
+    @SuppressWarnings("unchecked")
     @Override
     public SimpleVersionedSerializer<Committable> getCommittableSerializer() {
-        return CommittableSerializer.INSTANCE;
+        return new CommittableSerializer(
+                fileCommitSerializer(), (SimpleVersionedSerializer<Object>) logCommitSerializer());
     }
 
     @Override
-    public GlobalCommittableSerializer<LogCommT> getGlobalCommittableSerializer() {
-        ManifestCommittableSerializer fileCommSerializer =
-                new ManifestCommittableSerializer(
-                        project(rowType, partitions), project(rowType, keys), rowType);
-        SimpleVersionedSerializer<LogCommT> logCommitSerializer = new NoOutputSerializer<>();
-        return new GlobalCommittableSerializer<>(logCommitSerializer, fileCommSerializer);
+    public ManifestCommittableSerializer getGlobalCommittableSerializer() {
+        return new ManifestCommittableSerializer(
+                project(rowType, partitions), project(rowType, keys), rowType);
```

Comment:
`new ManifestCommittableSerializer(project(rowType, partitions), project(rowType, keys), rowType)` can be replaced with `new ManifestCommittableSerializer(partitionType, KeyType, ValueType)` now